openupgrade_scripts/scripts/account_edi_ubl_cii/18.0.1.0/upgrade_analysis.txt

```diff
@@ -3,7 +3,7 @@
 account_edi_ubl_cii / res.company              / invoice_is_ubl_cii (boolean)  : DEL
 account_edi_ubl_cii / res.partner              / invoice_edi_format (False)    : NEW selection_keys: ['facturx', 'nlcius', 'ubl_a_nz', 'ubl_bis3', 'ubl_sg', 'xrechnung'], mode: modify
 account_edi_ubl_cii / res.partner              / is_peppol_edi_format (boolean): previously in module account_peppol
-account_edi_ubl_cii / res.partner              / peppol_eas (selection)        : selection_keys removed: [0212, 0215, 9901]
+account_edi_ubl_cii / res.partner              / peppol_eas (selection)        : selection_keys added: [0218, 0225, 0235, 0240], removed: [0212, 0215, 9901]
```

Comment:
There's something to do here, isn't it?

Reply:
What?

Reply:
Update the remove keys to null or other value.

Reply:
That was done long time ago. The **new** change is the **added** ones.